### PR TITLE
alts: add ComputeEngineChannelBuilder to interop test

### PR DESCRIPTION
**Pull request by jiangtaoli2016**
Add compute_engine_credentials as custom_credentials_type option.

### CODE REVIEW

File: interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java [hunk 1]
```diff
@@ -373,6 +374,10 @@ protected ManagedChannel createChannel() {
           && customCredentialsType.equals("google_default_credentials")) {
         return GoogleDefaultChannelBuilder.forAddress(serverHost, serverPort).build();
       }
+      if (customCredentialsType != null
+          && customCredentialsType.equals("compute_engine_credentials")) {
```
Comment (apolcyn) [inline interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java]: sorry, mind changing to `compute_engine_channel_creds` ?

`compute_engine_credentials` is very similar to the `compute_engine_creds` test case, and it also may look like it corresponds to compute engine "call creds".

Comment (jiangtaoli2016) [reply interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java]: Done.

**Comment by jiangtaoli2016** [comment]
@apolcyn revised. PTAL.

**Comment by dapengzhang0** [comment]
A side note to @jiangtaoli2016: Could you please add an one paragraph README in examples/example-alts in what environment and what settings are needed so that external users can use alts channel?  #5130

**Comment by sanjaypujare** [comment]
I second @dapengzhang0's suggestion. Good to have a README that's enough for a user to execute the example

**Comment by jiangtaoli2016** [comment]
@dapengzhang0 @sanjaypujare I will open an email thread to discuss.  Thanks!

**Pull request merged**